libs/sdk-ui-kit/src/Dialog/typings.ts

```diff
@@ -11,6 +11,9 @@ export interface IDialogBaseProps {
     children?: React.ReactNode;
     className?: string;
     displayCloseButton?: boolean;
+    closeButtonAccessibilityConfig?: {
```

Comment:
It would be better to have `accessibilityConfig` prop on the dialog and inside of it have `closeButton` prop that would be typed to this object. You could end up with multiple `xxxAccessibilityConfig` props in the top level, polluting the props.

Reply:
will change